sai_p4/instantiations/google/minimum_guaranteed_sizes.p4

```diff
@@ -42,19 +42,28 @@
 // The maximum sum of weights across all wcmp groups.
 #define WCMP_GROUP_SELECTOR_MAX_SUM_OF_WEIGHTS_ACROSS_ALL_GROUPS 65536
 
+// The size semantics for WCMP group selectors.
+//#define WCMP_GROUP_SELECTOR_SIZE_SEMANTICS "SUM_OF_WEIGHTS"
+
 // The maximum sum of weights for each wcmp group.
+//#define WCMP_GROUP_SELECTOR_MAX_GROUP_SIZE 512
 #define WCMP_GROUP_SELECTOR_MAX_SUM_OF_WEIGHTS_PER_GROUP 256
 
+// The max weight of an individual member when using the SUM_OF_MEMBERS size
+// semantics.
+#define WCMP_GROUP_SELECTOR_MAX_MEMBER_WEIGHT 4096
 // -- ACL Table sizes ----------------------------------------------------------
+// Some switches allocate table sizes in powers of 2. However:
+// - They may reserve 1 entry in each ACL table for atomic operations. See
+//   b/233261593#comment2 for more info.
```

Comment:
Comments of this form b/number need to be removed.

Reply:
@kishanps , i have removed the comment as you suggested. 
Thanks